### PR TITLE
889176: Updated Review changes for UG documentation

### PR DESCRIPTION
**Pull request by SwethavasanthSF3977**

### CODE REVIEW

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ documentation: ug
 
 # Header and Footer support in .NET MAUI Autocomplete (SfAutocomplete)
 
-We can provide Header and Footer view in the dropdown in [`SfAutocomplete`] by enabling [`ShowDropDownHeaderView`] and [`ShowDropDownFooterView`]. 
+We can provide Header and Footer view in the dropdown in [SfAutocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) by enabling [ShowDropDownHeaderView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_ShowDropdownHeaderView) and [ShowDropDownFooterView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_ShowDropdownFooterView).
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Header-and-Footer.md:12]: ShowDropDownHeaderView => ShowDropdownHeaderView

Change the property names like above for all the properties.

File: MAUI/Radio-Button/Visual-Customization.md [hunk 2]
```diff
@@ -123,26 +130,30 @@ The `LineBreakMode` allows you to wrap or truncate the text. The default value o
 {% tabs %}
 {% highlight xaml %}
 
-    <syncfusion:SfRadioButton x:Name="RadioButton" IsChecked="True" WidthRequest="200" LineBreakMode="WordWrap" Text="The LineBreakMode allows you to wrap or truncate the text."></syncfusion:SfRadioButton>
+    <syncfusion:SfRadioButton   x:Name="RadioButton" 
+				IsChecked="True" 
+				WidthRequest="200" 
+				LineBreakMode="WordWrap" 
+				Text="The LineBreakMode allows you to wrap or truncate the text."></syncfusion:SfRadioButton>
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radio-Button/Visual-Customization.md:137]: Add this tag in next line </syncfusion:SfRadioButton>

File: MAUI/Autocomplete/Header-and-Footer.md (hunk 1 above)
Comment (SwethavasanthSF3977) [reply MAUI/Autocomplete/Header-and-Footer.md]: Changed the property name

File: MAUI/Radio-Button/Visual-Customization.md (hunk 2 above)
Comment (SwethavasanthSF3977) [reply MAUI/Radio-Button/Visual-Customization.md]: Added changes

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**